examples/notebooks/PoC_EODC.ipynb

```diff
@@ -15,7 +15,7 @@
    "source": [
     "import logging\n",
     "import openeo\n",
-    "from openeo.auth.auth_bearer import BearerAuth\n",
+    "from openeo.rest.auth.auth import BearerAuth\n",
```

Comment:
good catch, but this whole notebook looks completely outdated
e.g. a user should not have to import `BearerAuth` in the first place